packages/connect-plugin-ethereum/package.json

```diff
@@ -18,13 +18,14 @@
         "Ethereum"
     ],
     "sideEffects": false,
-    "publishConfig": {
-        "main": "lib/index.js"
-    },
     "main": "src/index.ts",
     "files": [
         "lib/"
     ],
+    "exports": {
+        "node": "./lib/index.js",
+        "default": "./libESM/index.js"
+    },
```

Comment:
This is following the approach describes here https://webpack.js.org/guides/package-exports/#providing-commonjs-and-esm-version-stateless